sdk/messaging/azeventhubs/internal/links.go

```diff
@@ -33,7 +32,7 @@ type LinkWithID[LinkT AMQPLink] struct {
 // (for unit testing only)
 type LinksForPartitionClient[LinkT AMQPLink] interface {
 	RecoverIfNeeded(ctx context.Context, partitionID string, lwid *LinkWithID[LinkT], err error) error
-	Retry(ctx context.Context, eventName log.Event, operation string, partitionID string, retryOptions exported.RetryOptions, fn func(ctx context.Context, lwid LinkWithID[LinkT]) error) error
+	Retry(ctx context.Context, eventName azlog.Event, operation string, partitionID string, retryOptions exported.RetryOptions, fn func(ctx context.Context, lwid LinkWithID[LinkT]) error) error
```

Comment:
I don't mind it, but why did this need to change?

Reply:
hmm yes it consolidated the duplicate imports